pkgs/top-level/python-packages.nix

```diff
@@ -1159,6 +1159,28 @@ in {
     };
   };
 
+  argcomplete = buildPythonPackage rec {
```

Comment:
Thanks! It's been a while since I last contributed a python library. Just pushed a new commit for this.